app/assets/stylesheets/mixins/_earmark.scss

```diff
@@ -0,0 +1,20 @@
+@mixin earmark($color, $size) {
+  &:before {
+    $border-transparent: $size solid rgba(0, 0, 0, 0);
+    $border-colored: $size solid $color;
+
+    border-bottom: $border-transparent;
+    border-left: $border-colored;
+    border-right: $border-transparent;
+    border-top: $border-colored;
+
+    height: $size;
```

Comment:
Properties should be ordered border-bottom, border-left, border-right, border-top, content, display, height, left, position, top, width